app/services/search/results/base_builder.rb

```diff
@@ -0,0 +1,17 @@
+module Search
+  module Results
+    class BaseBuilder
+      def initialize(record)
+        @record = record
+      end
+
+      def autocomplete_hint
+        'Not implement yet'
```

Comment:
i think u should add a raise 'NotImplemented', what do you think?

Reply:
sounds good